themes/mongoid/page.html

```diff
@@ -1,5 +1,13 @@
 {%- extends "layout.html" -%}
 
+{%- block htmltitle %}
+  {%- if not embedded and docstitle %}
+    {%- set titlesuffix = " &mdash; "|safe + docstitle|e + " " + release|e %}
+  {%- else %}
+    {%- set titlesuffix = "" %}
+  {%- endif -%}
```

Comment:
Do we need a `{%- endblock %}` here?

Reply:
lol, yes and we need the rest of this block :headdesk: